src/app/utils/dateFormatter.ts

```diff
@@ -24,13 +24,42 @@ const formatDistanceLocale = {
   almostXYears: '{{count}}y',
 }
 
-export const formatDistance = (token: keyof typeof formatDistanceLocale, count: string) =>
-  formatDistanceLocale[token].replace('{{count}}', count)
+const formatDistance = (
+  token: keyof typeof formatDistanceLocale,
+  count: string,
+  options: {
```

Comment:
What is a use case for these options in this PR ?

Reply:
What's a decision here? Do we use these options or not ?

Reply:
Personally I would leave it, just in case we get some product decision later, to add the suffix back.